### DISCUSSION TITLE
Laravel request: have all classes when in development mode again

### DISCUSSION THREAD

**Discussion by theoreaves** [tailwindlabs/tailwindcss#6973] [Help]
Hello,

I use laravel with tailwindcss.

Is there any "easy" way to have all tailwind classes available when you are in development mode?  I know that before, when I ran "npm run dev", I had them all.  It seemed to only limit it down when you did "npm run prod".  But now, I'm not getting them regardless.  

Or is there a way to have them all list in PHP Storm?

**Comment by Martleby** [comment]
Not in TW 3, as it is jit mode only. If it could, you'd end up with 300 mb + of CSS...

I think we're waiting for jet brains to come up with a solution.
